geolocator_platform_interface/pubspec.yaml

```
@@ -3,13 +3,13 @@ description: A common platform interface for the geolocator plugin.
 homepage: https://github.com/baseflow/flutter-geolocator
 # NOTE: We strongly prefer non-breaking changes, even at the expense of a
 # less-clean API. See https://flutter.dev/go/platform-interface-breaking-changes
-version: 3.0.1
+version: 3.0.2
```

Update to indicate the breaking change:
```suggestion
version: 4.0.0
```